src/pages/TeamPage/index.js

```diff
@@ -285,67 +284,65 @@ function TeamPage() {
-                                <Popup
-                                  Name={item.Name || "Awaiting Member's Name"}
-                                  Content={
-                                    (item.Content.length && item.Content) ||
-                                    "Awaiting content from team member"
+                      return (
+                        <Grid item xs={6} sm={6} md={3}>
+                          <Tippy
+                            animation="fade"
+                            interactive="true"
+                            duration={[500, 0]}
+                            placement={
+                              window.screen.availWidth < 650
+                                ? "bottom"
+                                : "right"
+                            }
+                            content={
+                              <Popup
+                                Name={item.Name || "Awaiting Member's Name"}
+                                Content={
+                                  (item.Content.length && item.Content) ||
+                                  "Awaiting content from team member"
+                                }
+                                linkedin={item.Linkedin}
+                                twitter={item.Twitter}
+                              />
+                            }
+                          >
+                            <div>
+                              <div
+                                className={`${classes.team_cardDetails} card-details`}
+                              >
+                                <img
+                                  className={
+                                    !isActive
+                                      ? `${classes.team_cardImg} img-hover`
+                                      : `${classes.team_mobileCardImg} img-hover`
                                   }
-                                  linkedin={item.Linkedin}
-                                  twitter={item.Twitter}
+                                  src={item.Photo}
+                                  alt={item.Name.substring(
+                                    0,
+                                    item.Name.indexOf(" ")
+                                  )}
                                 />
-                              }
-                            >
-                              <div>
-                                <div
-                                  className={`${classes.team_cardDetails} card-details`}
+                                <Typography
+                                  variant="body1"
+                                  className={classes.team_cardTitle}
+                                  style={
+                                    !isActive ? {} : { textAlign: "center" }
+                                  }
+                                >
+                                  {item.Name}
+                                </Typography>
```

Comment:
Recommend earlier [ResponsiveTypography suggestion](https://github.com/navgurukul/bhanwari-devi/pull/290#discussion_r853781939):
```
function ResponsiveTypography(props) {
  return <Typography textAlign={{xs: "center", sm: "left"}} {...props}/>
}
```
```suggestion
                                <ResponsiveTypography
                                  variant="body1"
                                  className={classes.team_cardTitle}
                                >
                                  {item.Name}
                                </ResponsiveTypography>
```

Reply:
What about sx that you've mentioned here [ResponsiveTypography suggestion](https://github.com/navgurukul/bhanwari-devi/pull/290#discussion_r853781939)

Reply:
The component there had an `sx` property so I kept it, but here it does not so I didn't include it.  The `ResponsiveTypography` component allows you to remove the common alignment style from each `Typography` one that you're converting so you'd remove something like this:
```
                                                  align={isActive ? "center" : "left"}
```
And then keep all of the other specified properties.